examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -57,7 +57,7 @@
     "HugeCTR is a GPU-accelerated recommender framework designed to distribute training across multiple GPUs and nodes and estimate Click-Through Rates (CTRs).<br>\n",
```

Comment:
<p>I think we should remove the information about Solver, Optimizer, Etc. We will not be able to keep them uptodate in the example for HugeCTR Team. Instead, we should reference the HugeCTR documentation: https://nvidia-merlin.github.io/HugeCTR/main/hugectr_user_guide.html# and API Documentation: https://nvidia-merlin.github.io/HugeCTR/main/api/index.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>removed, that is a good point</p>